pypeit/core/wavecal/wvutils.py

```diff
@@ -655,21 +724,34 @@ def wavegrid(wave_min, wave_max, dwave, spec_samp_fact=1.0, log10=False):
             spec_samp_fact are pixels.
 
     Returns:
-        `numpy.ndarray`_: Wavelength grid in Angstroms (i.e. log10 even
-        if log10 is requested)
+        :obj:`tuple`: Returns two `numpy.ndarray`_ objects and a float:
```

Comment:
Description needs to be on a new line, I think..

Reply:
fixed.